viewer/src/config.rs

```diff
@@ -0,0 +1,25 @@
+use std::collections::HashMap;
+use config::builder::DefaultState;
+use config::{Config, ConfigBuilder};
+use serde::{Deserialize, Serialize};
+
+pub fn config_builder() -> ConfigBuilder<DefaultState> {
+	Config::builder()
+		.add_source(config::File::new(
+			"tag_names.js",
```

Comment:
I'd rename this to config.js or something more generic for when other config options get added, and leave a note about it in the README